Resources/Private/Solr/configsets/ext_solr_13_0_0/conf/solrconfig.xml

```diff
@@ -4,15 +4,6 @@
 
 	<abortOnConfigurationError>${solr.abortOnConfigurationError:true}</abortOnConfigurationError>
 
-	<lib dir="${solr.install.dir:../../../..}/modules/extraction/lib/" regex=".*\.jar" />
```

Comment:
We should increase the config version, so that the config version could be checked by the reports module

IMHO it also could make sense to use branch 13.1 to indicate there is a change that requires some additional steps

Reply:
ok, as by using the script most users won't see any difference.